corehq/apps/geospatial/static/geospatial/js/geospatial_map.js

```diff
@@ -828,7 +365,7 @@ hqDefine("geospatial/js/geospatial_map", [
             // Hide the datatable rows but not the pagination bar
             $('.dataTables_scroll').hide();
 
-            if (xhr.responseJSON.aaData.length && map) {
```

Comment:
just confirming that this change means we are not ensuring there is a map anymore?